cpp/src/gandiva/precompiled/CMakeLists.txt

```diff
@@ -38,10 +50,13 @@ foreach(SRC_FILE ${PRECOMPILED_SRCS})
   add_custom_command(
     OUTPUT ${BC_FILE}
     COMMAND ${CLANG_EXECUTABLE}
+            ${PLATFORM_CLANG_OPTIONS}
             -DGANDIVA_IR
-            -std=c++11 -emit-llvm
+            -DNDEBUG # DCHECK macros not implemented in precompiled code
```

Comment:
is this required ? I modified arrow/logging.h to no-op all the DCHECK macros if GANDIVA_IR is enabled.

Reply:
This is a rebase artifact then, I will remove